### PR TITLE
[READY] RELEASE THE DRAGONS!!

### PR DESCRIPTION
**Pull request by Strazyplus**
### **[Attribution-NonCommercial-ShareAlike 3.0 Unported (CC BY-NC-SA 3.0)](https://creativecommons.org/licenses/by-nc-sa/3.0/)**

1595361337 miz-dee_okay-export

## About The Pull Request
Commissioned artist: deviantart.com/mizartz
Vorebellies: @ThighHighs

Though a commission with the talented Mizartz we have created a new subset of borgs for all to have, instead of being a dog the robot is a dragon or in this case a drake. the modules include Medical, Security, Mining, Peace, Engineering, and Janitor variants. in addition of  providing new borgshells we have also included plushie variants that can be obtained via plushie crate or via loadout and come in security and medical. 

janitor
mining
peace
security
engineering
medic
ca2rd

Drakeborg and drake plushies fall under [CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)
Share — copy and redistribute the material in any medium or format, Adapt — remix, transform, and build upon the material, however NonCommercial use — You may not use the material for commercial purposes.

## Why It's Good For The Game

Brings new subset of robots for borg players as well cute plushies for people to squeeze.

## Changelog
:cl:
add: Added drake drakeborg
imageadd: added drakeborg
add: Added drake drakeplushies
Code: Added drakeplush to item code
[CC BY-NC-SA 3.0](https://creativecommons.org/licenses/by-nc-sa/3.0/)
add: Added CC BY-NC-SA 3.0 license details to icon/mob/cyborg
rebuild of Drakeborgs
downstream compatible with citidel
adds drakeborg
adds drakeplushies to loadout.

/:cl:

### CODE REVIEW

File: tgstation.dme [hunk 1]
```diff
@@ -3601,8 +3601,8 @@
 #include "modular_skyrat\code\game\objects\items\circuitboards\computer_circuitboards.dm"
 #include "modular_skyrat\code\game\objects\items\circuitboards\machine_circuitboards.dm"
 #include "modular_skyrat\code\game\objects\items\devices\multitool.dm"
-#include "modular_skyrat\code\game\objects\items\devices\ttsdevice.dm"
```
Comment (Strazyplus) [inline tgstation.dme:3604]: ChillyEmbarrassedAnhinga-small
why? don't ask me why github does what the fuck it wants to do.

File: tgstation.dme [hunk 2]
```diff
@@ -3601,8 +3601,8 @@
...
 #include "modular_skyrat\code\game\objects\items\devices\traitordevices.dm"
+#include "modular_skyrat\code\game\objects\items\devices\ttsdevice.dm"
```
Comment (Strazyplus) [inline tgstation.dme:3605]: ChillyEmbarrassedAnhinga-small
Why? no idea. I didn't move it.

**Comment by BlackMajor** [comment]
Image of Derg

Your janitor drake bellyup sprites have the wrong eye colour and are uh.. offset eyes.

**Comment by Strazyplus** [comment]
image
fixed

**Comment by Strazyplus** [comment]
PR is complete, 100% bug fixing if anyone finds anything wrong.

**Comment by BlackMajor** [comment]
I mean if you wanted to go all the way ideally you'd also have ratvar eye overlays just for consistency (rip clockwork cult)

**Comment by Strazyplus** [comment]
> I mean if you wanted to go all the way ideally you'd also have ratvar eye overlays just for consistency (rip clockwork cult)

wait where are those sprites stored at? I can make glitchy eyes ez peezy

**Comment by BlackMajor** [comment]
> > I mean if you wanted to go all the way ideally you'd also have ratvar eye overlays just for consistency (rip clockwork cult)
> 
> wait where are those sprites stored at? I can make glitchy eyes ez peezy

When borg's get ratvar'd they get a different eye overlay, borgname_e_r, and flavourtext
These are generally just blazing bronze/gold eye overlays.
I just used a batch filter when porting borg sprites that didn't have them, but hey, if you want to go the extra mile and have the ratvar overlays be animated, that's some work to do.

**Comment by KasparoVy** [comment]
As mentioned on the last PR the walking animation's confusing but the sprites are bloody awesome. Like taking one step, pausing then taking another step with the same 'side' of legs, as if limping

**Comment by Strazyplus** [comment]
> As mentioned on the last PR the walking animation's confusing but the sprites are bloody awesome. Like taking one step, pausing then taking another step with the same 'side' of legs, as if limping

borgs are slow in game. but I do see what you mean but I believe its less noticeable in-game.

**Comment by KasparoVy** [comment]
@Strazyplus I primarily focus on the actual animations under checks before commenting so I don't end up with my foot in my mouth 😄 I understand about the GIFs.

**Comment by BlackMajor** [comment]
Headlight overlay for drakesec and drakeeng are scuffed and backwards

**Comment by Strazyplus** [comment]
> Headlight overlay for drakesec and drakeeng are scuffed and backwards

fixed

File: modular_skyrat/code/modules/client/loadout/__donator.dm [hunk 3]
```diff
@@ -21,3 +21,15 @@
 	category = SLOT_W_UNIFORM
 	path = /obj/item/clothing/under/custom/blutigen_undergarment
 	ckeywhitelist = list("random516")
+
+/datum/gear/toy/plush/borgplush/securitydrake
+	name = "Drake Plushie"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/toy/plush/borgplush/securitydrake
+	ckeywhitelist = list("navier")
```
Comment (CameronWoof) [inline modular_skyrat/code/modules/client/loadout/__donator.dm:29]: undocumented change

File: modular_skyrat/code/modules/client/loadout/__donator.dm [hunk 4]
```diff
@@ -21,3 +21,15 @@
...
+
+/datum/gear/toy/plush/borgplush/doctordrake
+	name = "Drake Plushie"
+	category = SLOT_IN_BACKPACK
+	path = /obj/item/toy/plush/borgplush/doctordrake
+	ckeywhitelist = list("navier")
```
Comment (CameronWoof) [inline modular_skyrat/code/modules/client/loadout/__donator.dm:35]: undocumented change

File: modular_skyrat/code/modules/client/loadout/__donator.dm (hunk 3 above)
Comment (Strazyplus) [reply modular_skyrat/code/modules/client/loadout/__donator.dm:29]: Added to the changelog, sorry about that.

File: modular_skyrat/code/modules/client/loadout/__donator.dm (hunk 4 above)
Comment (Strazyplus) [reply modular_skyrat/code/modules/client/loadout/__donator.dm:35]: Added to the changelog, sorry about that.

**Comment by Linzolle** [comment]
Please be aware that it is possible that the artist's [ToS](https://gdoc.pub/doc/1BtppIsljSgQ_yKVTEy1aJCkqFujjqGMY74KGCWnw6J4) are not compatible with the AGPL 3 license.

**Review by Gandalf2k15 — APPROVED** [review]
Testmerge probation complete. Final checks complete.

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 5]
```diff
@@ -1,3 +1,4 @@
+// Drakeborg sprites by Mizartz. drakeborg are licensed under the CC BY-NC-SA 3.0 license.
```
Comment (Iamgoofball) [inline code/modules/mob/living/silicon/robot/robot_modules.dm:1]: THIS IS NOT HOW LICENSES WORK

Comment (Strazyplus) [reply code/modules/mob/living/silicon/robot/robot_modules.dm:1]: Absolute goofball.

**Pull request merged**